nbs/tsdataset.ipynb

```diff
@@ -54,7 +54,8 @@
     "#| export\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #64.        def from_data_directories(directories, static_df=None, sort_df=False, temporal_cols=[], id_col='unique_id', time_col='ds', target_col='y'):
</pre><p>Please rename the temporal_cols argument here to <code>exogs</code> or similar to make it clearer that these are only the exogenous features</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>